docs/reference/stacker_file.md

```diff
@@ -1,13 +1,15 @@
 # The `stacker.yaml` file
 
-When doing a `stacker build`, the behavior of stacker is specified by the yaml
-directives below. In addition to these, stacker allows variable substitions of
-several forms. For example, a line like:
+During a stacker build, the behavior of stacker is specified by yaml directives in a `stacker.yaml` file, as described in this document.
+
+## Substitutions and variables
+
+In the yaml directives, stacker allows variable substitions of several forms. For example, these three substition statements:
 
     $ONE ${{TWO}} ${{THREE:3}}
```

Comment:
Whoops, we missed this part when I updated the stacker.yaml reference - we do not support `$ONE` anymore, it is OK to have that in the run section but it'll be interpreted as a shell variable by the shell running in the run section. 
It will not be treated as a substitution, and if you run `stacker build --substitute ONE=1` on this example you will get an error.

please see https://github.com/project-stacker/stacker/pull/612 for my rewritten version of this in the source